Makefile

```diff
@@ -78,4 +78,8 @@ helm-dry: ## Helm Dry Install
 helm-install: ## Helm Install
 	@cd helm && helm install -f YataiService/values/postgres.yaml yatai-service YataiService
 helm-uninstall: ## Helm Uninstall
-	helm uninstall yatai-service
\ No newline at end of file
+	helm uninstall yatai-service
+
+# Kubernetes
+k8s: ## Monitoring BentoService on K8s with Prometheus
+	@cd docs/source/guides/configs && ./k8s.sh -d
```

Comment:
this is meant for BentoML developers only, right?  we should probably avoid putting user-facing scripts into this makefile. (the helm related things above should be moved out as well)

Reply:
That is correct. I will just update `DEVELOPMENT.md` instead.